docs/public-networks/reference/cli/options.md

```diff
@@ -4731,14 +4731,9 @@ tx-pool="legacy"
 
 Type of [transaction pool](../../concepts/transactions/pool.md) to use.
 Set to `layered` to use the layered transaction pool implementation.
-Set to `legacy` to opt out of the layered transaction pool.
+Set to `sequenced` or `legacy` to opt out of the layered transaction pool.
```

Comment:
if `legacy` is being removed from everywhere else, why not remove it from here also?

Reply:
otherwise maybe 
```suggestion
Set to `sequenced` (previously known as `legacy`) to opt out of the layered transaction pool.
```